doc/nrf/samples/tfm.rst

```diff
@@ -5,6 +5,8 @@ Trusted Firmware-M (TF-M) samples
 
 This section lists the available |NCS| samples and tests for :ref:`ug_tfm`, one of the :ref:`security` features of the |NCS|.
 
+You can start by checking the :ref:`tfm_hello_world`, which demonstrates how to add TF-M to an application.
```

Comment:
```suggestion
You can start by checking the :ref:`tfm_hello_world` that demonstrates how to add TF-M to an application.
```

Reply:
I'll stick to my nonrestrictive clause :)